Specs/Scene/ExpressionSpec.js

```diff
@@ -814,6 +814,24 @@ defineSuite([
         expect(expression.evaluate(undefined)).toEqual(false);
     });
 
+    it('evaluates abs function', function() {
```

Comment:
Put this on a roadmap since, without this, the tests are too fine-grained.  It could just be one or a few tests with a complex style.